pkg/splunk/enterprise/util.go

```diff
@@ -83,6 +94,7 @@ func GetRemoteStorageClient(client splcommon.ControllerClient, cr splcommon.Meta
 
 	scopedLog.Info("Creating the client", "volume", vol.Name, "bucket", bucket, "bucket path", prefix)
 
+	var err error
```

Comment:
nit: why this here now? wasn't it working before or did we change anything wrt error in this code?

Reply:
It's because I nested the place that originally declared `err` via a `:=` assignment.